requirements/git.txt

```diff
@@ -0,0 +1,54 @@
+# The dependencies in this file are currently built from git repositories, and
+# then stored in our private pyrepo. Each commented-out git URL points to the
+# source revision the following requirement line was built from.
+
+# To upgrade one of these dependencies:
+#
+# 1) Update the portion of the git URL immediately following the `@` to the
+#    target commit hash.
+#
+# 2) Update the requirement line immediately following the git URL to point to
+#    the version number that commit corresponds to.
+#
+# 3) Build a new wheel for the updated git URL using `pip wheel <git-url>`.
+#
+# 4) Upload that wheel to our private pyrepo, or if you don't have access,
```

Comment:
Do you want to add a note explaining that the wheels should go in the `wheelhouse/` folder on pyrepo?